File: capture_okky.js
const puppeteer = require('puppeteer');
const path = require('path');
const os = require('os');

async function captureOkky() {
    const desktopPath = path.join(os.homedir(), 'Desktop');
    const timestamp = new Date().toISOString().replace(/[:.]/g, '-').slice(0, -5);
    const screenshotPath = path.join(desktopPath, `okky_screenshot_${timestamp}.png`);
    
    const browser = await puppeteer.launch({ headless: true });
    const page = await browser.newPage();
    await page.setViewport({ width: 1920, height: 1080 });
    await page.goto('http://okky.kr', { waitUntil: 'networkidle2' });
    await page.screenshot({ path: screenshotPath });
    await browser.close();
    
    console.log(`Screenshot saved: ${screenshotPath}`);
}

captureOkky().catch(console.error);

File: capture_okky.py
from selenium import webdriver
from selenium.webdriver.chrome.options import Options
import os
from datetime import datetime
import time

def capture_okky():
    desktop_path = os.path.expanduser("~/Desktop")
    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
    screenshot_path = os.path.join(desktop_path, f"okky_screenshot_{timestamp}.png")
    
    chrome_options = Options()
    chrome_options.add_argument('--headless')
    chrome_options.add_argument('--window-size=1920,1080')
    
    driver = webdriver.Chrome(options=chrome_options)
    driver.get('http://okky.kr')
    time.sleep(3)
    driver.save_screenshot(screenshot_path)
    driver.quit()
    
    print(f"Screenshot saved: {screenshot_path}")
    return screenshot_path

if __name__ == "__main__":
    capture_okky()
